packages/taquito/src/estimate/rpc-estimate-provider.ts

```diff
@@ -167,6 +175,12 @@ export class RPCEstimateProvider extends OperationEmitter implements EstimationP
       totalStorage += 'originated_rollup' in result ? tx_rollup_origination_size : 0;
     });
 
+    const asSmartRollupOriginate = content as OperationContentsAndResultSmartRollupOriginate;
```

Comment:
Can you elaborate a bit what does line 178-182 do?

Reply:
removing casting
ts was complaining but i think i needed to refresh my ide since its not complaining anymore